build/lib/release.sh

```diff
@@ -383,7 +383,11 @@ EOF
           echo "COPY nsswitch.conf /etc/" >> "${docker_file_path}"
         fi
 
-        "${DOCKER[@]}" build ${docker_build_opts:+"${docker_build_opts}"} -q -t "${docker_image_tag}" "${docker_build_path}" >/dev/null
+	if [[ "${arch}" == "linux/s390x" || "${arch}" == "linux/ppc64le" ]]; then
+          DOCKER_CLI_EXPERIMENTAL=enabled "${DOCKER[@]}" buildx build --platform linux/"${arch}" --load ${docker_build_opts:+"${docker_build_opts}"} -q -t "${docker_image_tag}" "${docker_build_path}" >/dev/null
```

Comment:
why aren't we using buildx on all paltforms?
cc @dims @justaugustus @vinayakankugoyal

Reply:
it seems https://github.com/kubernetes/kubernetes/issues/97458 was also only solved for the specific list of platforms in the issue?